tests/lora/test_lora_layers_old_backend.py

```diff
@@ -2256,21 +2256,36 @@ def test_sdxl_1_0_last_ben(self):
         self.assertTrue(np.allclose(images, expected, atol=1e-3))
 
     def test_sdxl_1_0_fuse_unfuse_all(self):
-        pipe = DiffusionPipeline.from_pretrained("stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.float16)
```

Comment:
ideally these tests don't need to be changed. They should only run on GPU anyways

Reply:
I get GPU OOM when running it on GPU :/ I think that's why they are initially on CPU

Reply:
Do you still get GPU OOMs?

Reply:
Hmm that's weird SDXL should easily be run-able on a V100

Reply:
The model is not that big (only ~6GB in float16)

Reply:
This is now fixed thanks to https://github.com/huggingface/peft/pull/1017